pykickstart/commands/ostreecontainer.py

```diff
@@ -77,6 +83,12 @@ def _getParser(self):
         op.add_argument("--transport", version=F38, help="""
                         The transport; e.g. registry, oci, oci-archive.
                         The default is `registry`.""")
+        op.add_argument("--target-url", version=F38, help="""
```

Comment:
version refers to the Fedora version of Anaconda this command is added to, is it going to be backported all the way to F38 or is this going to be F40?

Reply:
Yeah changed to use 40